src/UI/Window.cpp

```diff
@@ -766,3 +766,64 @@ void cWindow::SetProperty(short a_Property, short a_Value, cPlayer & a_Player)
 
 
 
+
+
+void cWindow::ClearCraftingGrid(cPlayer & a_Player)
```

Comment:
Since these window changes are specific to crafting, they probably belong in `cSlotAreaCrafting`.